tests/ui-cargo/inherit_config/inner/clippy.toml

```diff
@@ -0,0 +1,4 @@
+too-many-lines-threshold = 3
+msrv = "1.1"
+# Will emit an error if uncommented, and revert to default config
```

Comment:
Extending this comment with the reason why an error will be emitted if uncommented would be very helpful (not having priority 0 AFAIK)